src/Components/ServiceSelectionScene/ServiceSelectionScene.tsx

```diff
@@ -677,7 +730,25 @@ export class ServiceSelectionScene extends SceneObjectBase<ServiceSelectionScene
     }
   }
 
-  private updateBody() {
+  private getGridItems(): SceneCSSGridItem[] {
+    return this.state.body.state.children as SceneCSSGridItem[];
+  }
+
+  private getVizPanel(child: SceneCSSGridItem) {
+    return child.state.body as VizPanel | undefined;
+  }
+
+  private runPanelQuery(child: SceneCSSGridItem) {
+    if (child && child.isActive) {
```

Comment:
If you need to test for child you should update the type in the function.

Reply:
Agreed, fixed